src/styles.css

```diff
@@ -0,0 +1,20 @@
+body {
```

Comment:
I think your site would look better if it took the whole width of the window. Instead of 
`width: 100%`
you could use:
`width: 100vw`
so that it fills the width of the browser window.